### DISCUSSION TITLE
server unknown argument: --mmproj

### DISCUSSION THREAD

**Discussion by TemporalAgent7** [ggml-org/llama.cpp#5967] [Q&A]
Hi,
I'm brand new to llama.cpp; just did a build on Windows (from the top of the master branch) and trying to run server.exe I'm seeing this output:

```
server.exe -m ggml-model-q4_k.gguf --mmproj mmproj-model-f16.gguf 
error: unknown argument: --mmproj
usage: server.exe [options]
```

The model and projection loads fine if I use llava-cli from the same local build.

Are there additional steps (configuration options, or different branches) needed to have multimodal support enabled in server?

Thank you!

**Comment by phymbert** [comment]
Hi, multimodal support has been temporarily dropped in #5882

**Reply by TemporalAgent7** [reply]
I see; do you know if we can run llava-cli as a sort-of server? Basically have it load the models just once and keep passing new images and prompts? I'm ok if it's stdin/stdout only, can wrap it in something else.

I guess the alternative is to sync before 5882, right?

**Reply by phymbert** [reply]
you can create a feature request. Afaik, there is none opened, but it will require some implementation effort on the server. 
Meanwhile, checkout ceca1ae
